Makefile

```diff
@@ -6,35 +6,35 @@ VERSION_NAME=v1.0.0
 lint:
 	black src/
 	flake8 src/
-.phony: lint
+.PHONY: lint
 
 run_app:
 	set -a && source .env && set +a && python src/main.py
-.phony: run_app
+.PHONY: run_app
 
 create_image:
 	docker build -t ${MODULE}:${VERSION_NAME} . -f docker/Dockerfile
-.phony: create_image
+.PHONY: create_image
 
 run_image:
-	docker run --device=/dev/ttyUSB0  --rm --env-file=./config.env ${MODULE}:${VERSION_NAME}
-.phony: run_image
+	docker run --device=/dev/ttyACM0 --rm --env-file=./.env ${MODULE}:${VERSION_NAME}
```

Comment:
why this change?

Reply:
`.env` because that's what we use in other modules. acm instead of usb because that is the name linux gives to the ST boards we are currently using